common/primitives/src/msa.rs

```diff
@@ -210,3 +210,36 @@ pub struct KeyInfoResponse<AccountId> {
 	/// The MSA associated with the `key`
 	pub msa_id: MessageSourceId,
 }
+
+#[cfg(test)]
```

Comment:
Simple tests of the scale decoding. Worthwhile? Perhaps not, but not bad examples of how to test it.